music21/freezeThaw.py

```diff
@@ -758,12 +758,6 @@ def teardownSerializationScaffold(self, streamObj=None):
         >>> st = freezeThaw.StreamThawer()
         >>> st.teardownSerializationScaffold(a)
         '''
-        def _fixId(innerEl):
-            if (innerEl.id is not None
-                    and common.isNum(innerEl.id)
-                    and innerEl.id > defaults.minIdNumberToConsiderMemoryLocation):
-                innerEl.id = id(innerEl)
-
```

Comment:
I think this will be a change in behavior if someone does this:
```
>>> from music21 import stream
>>> v = stream.Voice()
>>> v2 = stream.Voice()
>>> v2.id = v.id
>>> v.id
4569179616
>>> v2.id
4569179616
>>> freeze/thaw
>>> thawedVoice.id
4569179616
```
Is that tolerable? Or do we want to emit a warning when setting id when it's above the defaults.minimumToConsider...